modules/service/controllers/externalservice/populate/populate.go

```diff
@@ -57,7 +57,7 @@ func getSpec(es *riov1.ExternalService, os *objectset.ObjectSet) (v1.ServiceSpec
 		}
 		return v1.ServiceSpec{
 			Type:         v1.ServiceTypeExternalName,
-			ExternalName: fmt.Sprintf("%s.%s", service, ns),
+			ExternalName: fmt.Sprintf("%s.%s.svc.cluster.local", service, ns),
```

Comment:
Without this it doesn't scope to the cluster ?

Reply:
yes, it is a cname record so it will return $name.$namespace. Kubernetes use search path to resolve that but looks like it is not getting resolved for cname record.